### DISCUSSION TITLE
Custom Formatter for background color sets tooltip but not background color

### DISCUSSION THREAD

**Discussion by KingsGithub** [ghiscoding/aurelia-slickgrid#1291] [Q&A] [answered]
Hi,
I've written a custom formatter as follows
```
 BgColorFormatter = {
        exproStatus: function (row, cell, value, columnDef, dataContext) {
            let item = dataContext;
            let rtn = { text: value, removeClasses: 'cellBgColorFFC8CE cellBgColorF4AF85 cellBgColorFFC8CE', 
                            addClasses :'', toolTip:'' };
            let backgroundColor="";

            if(value==''||value is undefined) return rtn;

            if ( item.exproStatus  ==  "Withdrawn" || item.exproStatus  ==  "Outstanding"  ) backgroundColor = "cellBgColorFEEB9C";
            else if ( item.exproStatus  ==  "Expired"  )   backgroundColor = "cellBgColorFFC8CE";
            else if ( item.exproStatus  ==  "In Place"  )  backgroundColor = "cellBgColorC6EFCD";
            rtn.addClasses = backgroundColor; 
            rtn.toolTip = 'testing the formatting tooltip';
            console.log(rtn);
            return rtn;
        }
        .....other formatting functions....
}
```

**IN THE COLUMNDEFINITION:**
{id : 'exproStatus', name : 'Expropriation Status', field: 'exproStatus', type: FieldType.string, 
                formatter: this.BgColorFormatter.exproStatus, filterable: true, filter: { model: Filters.compoundInputText}, sortable: true, minWidth:20 },
                
 **IN THE GRID ON DISPLAY** the tooltip is clearly seen when hovering over the column but the color has not been set.
 **EDIT: HTML ON INSPECTION:**
` <div class="slick-cell l5 r5 cellBgColorFEEB9C" role="gridcell" tabindex="-1" aria-describedby="slickgrid_277111exproStatus" title="testing the formmatting tooltip">Outstanding</div>`

 The CSS Class  is available and works when applied to the `<aurelia-slickgrid>` component as in 
`<aurelia-slickgrid  css.bind="cssBackground"....></aurelia-slickgrid> where cssBackground is set in the view-model to 'cellBgColorC6EFCD'`

Any guidance would be appreciated. Thanks.
P.S I've also used the setCellCssStyles function and the colors also doesn't change.

**Comment by ghiscoding** [comment, marked as answer]
maybe use `!important` on your CSS color

**Reply by KingsGithub** [reply]
Thanks, will give that a try.
EDIT: It worked like a charm! Much appreciated!
